CONTRIBUTING.md

```diff
@@ -1,20 +1,24 @@
 # Contributing to NNCF
 
 Contributions are accepted in the form of:
+
 * Submitting issues against the current code to report bugs or request features
 * Extending NNCF functionality with important features (e.g. to address community requests, improve usability, implement a recently published compression algorithm, etc.)
 * Adding example scripts to showcase NNCF usage in real training pipelines and provide the means to reproduce the reported compression results
 * Providing recipes (specific NNCF configurations and training hyperparameters) to obtain state-of-the-art compression using NNCF for existing models
 * Adding well-defined patches that integrate NNCF into third-party repositories
-* Reducing performance overhead of NNCF compression by writing specialized CUDA kernels for compression operations or improving existing ones. 
+* Reducing performance overhead of NNCF compression by writing specialized CUDA kernels for compression operations or improving existing ones.
 
 The latter forms are accepted as pull requests from your own forks of the NNCF repository.
 
 Any contributions must not violate the repository's [LICENSE](./LICENSE) requirements.
 
 ## Installation
-### (Experimental) ONNXRuntime-OpenVINO
+
+## (Experimental) ONNXRuntime-OpenVINO
+
```

Comment:
Shouldn't it be `### (Experimental) ONNXRuntime-OpenVINO`? Otherwise there two same level header one after another

Reply:
Interesting but looks like it should not be here...  was added in https://github.com/openvinotoolkit/nncf/pull/1202

@alexsu52 @vshampor what do you think? Can i remove onnx instruction from CONTRIBUTING.md?
https://github.com/openvinotoolkit/nncf/pull/1996/commits/bbdefae7b90d5c88ed54df13cb9c47e1ff7b88e4

Reply:
Looks like we can remove it